[Bug] JsonUtils not support serialize java.time.LocalDate

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.


### Environment

Windows

### EventMesh version

master

### What happened

```
final Map<String, Object> content = new HashMap<>();
content.put("content", "testAsyncMessage");
 content.put("content", LocalDate.now());

JsonUtils.toJSONString(content)
```
throw exception


### How to reproduce

```
final Map<String, Object> content = new HashMap<>();
content.put("content", "testAsyncMessage");
 content.put("content", LocalDate.now());

JsonUtils.toJSONString(content)
```

### Debug logs

```
Exception in thread "main" org.apache.eventmesh.common.exception.JsonException: serialize to json error
	at org.apache.eventmesh.common.utils.JsonUtils.toJSONString(JsonUtils.java:66)
	at org.apache.eventmesh.grpc.GrpcAbstractDemo.buildEventMeshMessage(GrpcAbstractDemo.java:71)
	at org.apache.eventmesh.grpc.pub.eventmeshmessage.AsyncPublishInstance.main(AsyncPublishInstance.java:49)
Caused by: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Java 8 date/time type `java.time.LocalDate` not supported by default: add Module "com.fasterxml.jackson.datatype:jackson-datatype-jsr310" to enable handling (through reference chain: java.util.HashMap["content"])
	at com.fasterxml.jackson.databind.exc.InvalidDefinitionException.from(InvalidDefinitionException.java:77)
	at com.fasterxml.jackson.databind.SerializerProvider.reportBadDefinition(SerializerProvider.java:1300)
	at com.fasterxml.jackson.databind.ser.impl.UnsupportedTypeSerializer.serialize(UnsupportedTypeSerializer.java:35)
	at com.fasterxml.jackson.databind.ser.std.MapSerializer.serializeOptionalFields(MapSerializer.java:869)
	at com.fasterxml.jackson.databind.ser.std.MapSerializer.serializeWithoutTypeInfo(MapSerializer.java:760)
	at com.fasterxml.jackson.databind.ser.std.MapSerializer.serialize(MapSerializer.java:720)
	at com.fasterxml.jackson.databind.ser.std.MapSerializer.serialize(MapSerializer.java:35)
	at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider._serialize(DefaultSerializerProvider.java:480)
	at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.serializeValue(DefaultSerializerProvider.java:319)
	at com.fasterxml.jackson.databind.ObjectMapper._writeValueAndClose(ObjectMapper.java:4569)
	at com.fasterxml.jackson.databind.ObjectMapper.writeValueAsString(ObjectMapper.java:3822)
	at org.apache.eventmesh.common.utils.JsonUtils.toJSONString(JsonUtils.java:64)
	... 2 more
```

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

--- a/build.gradle
+++ b/build.gradle
@@ -458,6 +458,7 @@ subprojects {
             dependency "com.fasterxml.jackson.core:jackson-core:2.13.0"
             dependency "com.fasterxml.jackson.core:jackson-annotations:2.13.0"
             dependency "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.13.0"
+            dependency "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.13.0"
 
             dependency "org.asynchttpclient:async-http-client:2.12.0"
             dependency "org.apache.httpcomponents:httpclient:4.5.13"

--- a/eventmesh-common/build.gradle
+++ b/eventmesh-common/build.gradle
@@ -36,6 +36,7 @@ dependencies {
     api "com.fasterxml.jackson.core:jackson-databind"
     api "com.fasterxml.jackson.core:jackson-core"
     api "com.fasterxml.jackson.core:jackson-annotations"
+    api "com.fasterxml.jackson.datatype:jackson-datatype-jsr310"
 
     implementation "org.apache.httpcomponents:httpclient"
 

--- a/eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/JsonUtils.java
+++ b/eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/JsonUtils.java
@@ -34,6 +34,7 @@ import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
 
 /**
  * Json serialize or deserialize utils.
@@ -48,6 +49,7 @@ public class JsonUtils {
         OBJECT_MAPPER.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
         OBJECT_MAPPER.setDateFormat(new EventMeshDateFormat(Constants.DATE_FORMAT_DEFAULT));
         OBJECT_MAPPER.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+        OBJECT_MAPPER.registerModule(new JavaTimeModule());
     }
 
     /**

--- a/eventmesh-common/src/test/java/org/apache/eventmesh/common/utils/JsonUtilsTest.java
+++ b/eventmesh-common/src/test/java/org/apache/eventmesh/common/utils/JsonUtilsTest.java
@@ -44,7 +44,7 @@ public class JsonUtilsTest {
         jsonString = JsonUtils.toJSONString(map);
         Assert.assertEquals("{\"mxsm\":\"2\"}", jsonString);
 
-        Map<String,Object> maps = new HashMap<>();
+        Map<String, Object> maps = new HashMap<>();
         maps.put("mxsm", LocalDate.of(2013, 6, 28));
         jsonString = JsonUtils.toJSONString(maps);
         Assert.assertEquals("{\"mxsm\":\"2013-06-28\"}", jsonString);

--- a/eventmesh-common/src/test/java/org/apache/eventmesh/common/utils/JsonUtilsTest.java
+++ b/eventmesh-common/src/test/java/org/apache/eventmesh/common/utils/JsonUtilsTest.java
@@ -18,6 +18,7 @@
 package org.apache.eventmesh.common.utils;
 
 import java.nio.charset.StandardCharsets;
+import java.time.LocalDate;
 import java.util.Date;
 import java.util.GregorianCalendar;
 import java.util.HashMap;
@@ -42,6 +43,11 @@ public class JsonUtilsTest {
         map.put("mxsm", "2");
         jsonString = JsonUtils.toJSONString(map);
         Assert.assertEquals("{\"mxsm\":\"2\"}", jsonString);
+
+        Map<String,Object> maps = new HashMap<>();
+        maps.put("mxsm", LocalDate.of(2013, 6, 28));
+        jsonString = JsonUtils.toJSONString(maps);
+        Assert.assertEquals("{\"mxsm\":\"2013-06-28\"}", jsonString);
     }
 
     @Test

--- a/tools/dependency-check/known-dependencies.txt
+++ b/tools/dependency-check/known-dependencies.txt
@@ -75,6 +75,7 @@ jackson-annotations-2.13.0.jar
 jackson-core-2.13.0.jar
 jackson-databind-2.13.0.jar
 jackson-dataformat-yaml-2.13.0.jar
+jackson-datatype-jsr310-2.13.0.jar
 javassist-3.24.0-GA.jar
 javax.activation-1.2.0.jar
 javax.annotation-api-1.3.2.jar

